xbmc/network/WebServer.cpp

```diff
@@ -1035,10 +1035,43 @@ void CWebServer::ContentReaderFreeCallback(void *cls)
 #endif
 }
 
+// local helper
+static void MHDPanicHandler(void* unused, const char* file, unsigned int line, const char *reason)
+{
+  CLog::Log(LOGSEVERE, "MHD serious error: reason \"%s\" in file \"%s\" at line %ui", reason ? reason : "",
```

Comment:
Maybe prepend all MHD originated log messages with `CWebServer [MHD]:`? That way it's more consistent with the rest of the webserver logging.